### PR TITLE
added api name to config json file and added test for URL resolver

### PR DESCRIPTION
**Pull request by ashraf-km**
### All Submissions:

* [ ] Have you followed the guidelines in our [Contributing](../blob/master/contributing.md) document?
* [ ] Have you read and signed the automated Contributor's License Agreement?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### New Feature Submissions:

1. [ ] Does your submission pass tests?
2. [ ] Have you lint your code locally prior to submission?

### Changes to Core Features:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: src/lib/httpsClient.ts [hunk 1]
```diff
@@ -18,6 +18,10 @@ class HttpsClient {
     });
 
     req.on("error", (e) => {
+      console.log('exception: ', e);
+
+      // const RESOLVE_URL = `https://developer.paypay.ne.jp/develop/resolve?api_name=${}&code=${}&code_id=${}`
```
Comment (solancer) [inline src/lib/httpsClient.ts:23]: when there is an error it should print the console.log with the link.

code and code_id you will you get from the server response the API name you have to fetch from the conf method mapping

**Review by solancer — CHANGES_REQUESTED** [review]

File: src/lib/httpsClient.ts [hunk 2]
```diff
@@ -28,6 +28,10 @@ class HttpsClient {
...
+      // console.log(`This link should help you to troubleshoot the error: ${RESOLVE_URL}`)
```
Comment (shreyanshp) [inline src/lib/httpsClient.ts:34]: Why is this commented out?

**Review by shreyanshp — COMMENTED** [review]
@solancer is this PR WIP or completed?

**Comment by solancer** [comment]
Closing this PR since changes related to this was merged into master from PR #129

**Pull request closed without merging**